scripts/hail_batch/hgdp1kg_tobwgs_nfe/README.md

```diff
@@ -0,0 +1,9 @@
+# Perform pca and variant analysis on NFE samples
+
+This runs a Hail query script in Dataproc using Hail Batch in order to perform pca and variant frequency analysis on non-Finnish European (nfe) samples from the 1KG + HGDP dataset. To run, use conda to install the analysis-runner, then execute the following command:
+
+```sh
+analysis-runner --dataset ancestry \
+--access-level test --output-dir "gs://cpg-ancestry-temporary/1kg_hgdp_nfe/v0" \
```

Comment:
You won't be able to read from the `analysis` bucket using the `test` access level. You'd probably also want to keep the result in the `analysis` bucket, not in `temporary`?

Reply:
I changed the access-level to standard and the --outputdir to analysis